sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.netstandard2.0.cs

```diff
@@ -898,6 +898,7 @@ public static partial class BlobsModelFactory
         public static Azure.Storage.Blobs.Models.BlockInfo BlockInfo(byte[] contentHash, byte[] contentCrc64, string encryptionKeySha256) { throw null; }
         public static Azure.Storage.Blobs.Models.BlockInfo BlockInfo(byte[] contentHash, byte[] contentCrc64, string encryptionKeySha256, string encryptionScope) { throw null; }
         public static Azure.Storage.Blobs.Models.BlockList BlockList(System.Collections.Generic.IEnumerable<Azure.Storage.Blobs.Models.BlobBlock> committedBlocks = null, System.Collections.Generic.IEnumerable<Azure.Storage.Blobs.Models.BlobBlock> uncommittedBlocks = null) { throw null; }
+        public static Azure.Storage.Blobs.Models.ContainerSubmitBatchResult ContainerSubmitBatchResult(string contentType, System.IO.Stream body) { throw null; }
```

Comment:
Can we hold off merging this?  The swagger readme transforms to hide stuff like this are in the stg76base branch, which I will merge next week.  https://github.com/Azure/azure-sdk-for-net/pull/18068